parseutil/parseutil_test.go

```diff
@@ -487,3 +487,85 @@ func Test_ParseIntSlice(t *testing.T) {
 		}
 	}
 }
+
+func equalStringSlice(a, b []string) bool {
```

Comment:
There's a helper for this in strutil if you want.

Reply:
The one similar helper that I have found is `EquivalentSlices`. It doesn't doesn't actually check strict equality. Instead, it ensures that the sets of values are the same independent of the order. It ultimately performs a sort under the hood prior to looping and comparing each value. Were you thinking of a different function that I might have missed?

Reply:
Nope, just mistaken.

Reply:
Bummer, thanks for the suggestion though! I'll keep that package in mind for the future.